src/webrtc/groupCallEventHandler.ts

```diff
@@ -50,14 +50,14 @@ export class GroupCallEventHandler {
 
         const groupCallId = event.getStateKey();
 
-        const callType = content.type;
+        const callType = content["m.type"];
```

Comment:
Yup, thanks for changing these. Was in a rush to push things out last night so you could continue working.